File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-use-word-dialog-boxes-in-hidden-mode_1.vb
        Dim dialog As Word.Dialog = Application.Dialogs.Item(Word.WdWordDialog.wdDialogFilePageSetup)

        ' Set the properties of the dialog box.
        ' ControlChars.Quote() is used to represent the symbol for inches.
        With dialog
            .PageWidth = 3.3 & ControlChars.Quote
            .PageHeight = 6 & ControlChars.Quote
            .TopMargin = 0.71 & ControlChars.Quote
            .BottomMargin = 0.81 & ControlChars.Quote
            .LeftMargin = 0.66 & ControlChars.Quote
            .RightMargin = 0.66 & ControlChars.Quote
            .HeaderDistance = 0.28 & ControlChars.Quote
            .Orientation = Word.WdOrientation.wdOrientPortrait
            .DifferentFirstPage = False
            .FirstPage = 0
            .OtherPages = 0

            ' Apply these settings only to the current selection with this line of code:
            .ApplyPropsTo = 3

            ' Apply the settings.
            .Execute()
        End With

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-use-word-dialog-boxes-in-hidden-mode_1.cs
            dynamic dialog = Application.Dialogs[Word.WdWordDialog.wdDialogFilePageSetup];
            dialog.PageWidth = "3.3\"";
            dialog.PageHeight = "6\"";
            dialog.TopMargin = "0.71\"";
            dialog.BottomMargin = "0.81\"";
            dialog.LeftMargin = "0.66\"";
            dialog.RightMargin = "0.66\"";
            dialog.HeaderDistance = "0.28\"";
            dialog.Orientation = "0";
            dialog.DifferentFirstPage = "0";
            dialog.FirstPage = "0";
            dialog.OtherPages = "0";

            // Apply these settings only to the current selection with this line of code:
            dialog.ApplyPropsTo = "3";

            // Apply the settings.
            dialog.Execute(); 